sdk/core/azure-core/src/main/java/com/azure/core/http/policy/RetryPolicy.java

```diff
@@ -17,19 +19,22 @@
 
 /**
  * A pipeline policy that retries when a recoverable HTTP error occurs.
+ * @see RetryPolicyOptions
  */
 public class RetryPolicy implements HttpPipelinePolicy {
 
+    private final ClientLogger logger = new ClientLogger(RetryPolicy.class);
+
     private static final String RETRY_AFTER_MS_HEADER = "retry-after-ms";
 
-    private final ClientLogger logger = new ClientLogger(RetryPolicy.class);
-    private final RetryStrategy retryStrategy;
+    private final RetryPolicyOptions retryPolicyOptions;
 
     /**
-     * Creates a default {@link ExponentialBackoff} retry policy.
+     * Creates {@link RetryPolicy} with default {@link ExponentialBackoff} as {@link RetryStrategy}and use
+     * 'retry-after-ms' in {@link HttpResponse} header for calculating retry delay.
      */
     public RetryPolicy() {
-        this(new ExponentialBackoff());
+        this(new RetryPolicyOptions(new ExponentialBackoff(), RETRY_AFTER_MS_HEADER, ChronoUnit.MILLIS));
```

Comment:
I don't think the default should use `retry-after-ms` for the header, as far as I know this is only used by AppConfiguration.

Reply:
Updated AppConfiguration with this header